### PR TITLE
Adds a new submap for people to die on.

### PR DESCRIPTION
**Pull request by Naxodile**
The ship is a small (50~ x 20~ x 2) exploration vessel with enough refits and in-flight repairs that it can only be described as 'technically functional.' The submap job itself is supposed to be a sort of minor antag made to cause conflict with the salvage crews directly or by angrily complaining over the holocomms system.
At some point in the future I'll work on having ship weapons be unique to the ship itself, which'll make this into an actual pirate ship.

### CODE REVIEW

File: maps/away/noctis/noctis.dm [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
...',3\],\[6,'O',1\]\],\[\[2,'O',1\],\[4,'I',1\]\],\[\[2,'O',1\],\[8,'I',1\]\],\[\[2,'A',1\],\[7,'A',2\]\],\[\[2,'A',3\],\[8,'A',1\]\],\[\[3,'O',1\],\[6,'I',1\]\],\[\[3,'O',2\],\[6,'I',3\]\],\[\[3,'A',1\],\[6,'A',1\]\],\[\[4,'A',1\],\[8,'A',2\]\],\[\[5,'O',1\],\[13,'I',4\]\],\[\[6,'O',1\],\[7,'I',2\]\],\[\[6,'A',2\],\[7,'A',1\]\],\[\[7,'I',1\],\[10,'O',1\]\],\[\[8,'I',2\],\[9,'O',1\]\],\[\[11,'A',1\],\[12,'A',1\]\],\[\[11,'A',1\],\[18,'A',1\]\],\[\[12,'O',1\],\[13,'I',1\]\],\[\[12,'O',2\],\[13,'I',2\]\],\[\[12,'O',3\],\[13,'I',3\]\],\[\[12,'A',2\],\[13,'A',1\]\],\[\[13,'O',1\],\[14,'I',1\]\],\[\[13,'O',1\],\[16,'I',1\]\],\[\[13,'A',2\],\[14,'A',1\]\],\[\[14,'O',1\],\[15,'I',1\]\],\[\[14,'A',2\],\[15,'A',1\]\],\[\[15,'O',1\],\[16,'I',4\]\],\[\[15,'A',2\],\[16,'A',1\]\],\[\[16,'O',1\],\[17,'I',3\]\],\[\[16,'A',2\],\[17,'A',1\]\],\[\[18,'O',1\],\[21,'I',1\]\],\[\[18,'O',2\],\[22,'I',1\]\],\[\[18,'O',3\],\[23,'I',1\]\],\[\[18,'A',2\],\[21,'A',1\]\],\[\[19,'I',1\],\[20,'O',1\]\],\[\[19,'A',1\],\[20,'A',2\]\],\[\[20,'I',1\],\[21,'O',1\]\],\[\[20,'I',3\],\[22,'O',1\]\],\[\[20,'I',5\],\[23,'O',1\]\],\[\[20,'A',1\],\[23,'A',2\]\],\[\[21,'A',2\],\[22,'A',1\]\],\[\[22,'A',2\],\[23,'A',1\]\]\]}"}
```
Comment (Glloyd) [inline maps/away/noctis/noctis.dm]: I'm just going to assume you know what you're doing here

File: maps/away/noctis/noctis_shuttle.dm [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/datum/shuttle/autodock/overmap/raptor
```
Comment (Glloyd) [inline maps/away/noctis/noctis_shuttle.dm]: I can add a docking port for this on Nerva once it's merged, or add landmarks around Nerva for it.

File: code/modules/mob/living/carbon/human/spawners.dm [hunk 3]
```diff
@@ -82,21 +88,16 @@
...
-			var/obj/item/weapon/arrow/rod/R = new(H)
 
 			if(T)
+				var/obj/item/weapon/arrow/rod/R = new(H)
+				H.set_dir(GLOB.reverse_dir[dir])
 				H.loc = T
 				H.anchored = 1
 				H.pinned += R
 
-	if(viruses)
```
Comment (Glloyd) [inline code/modules/mob/living/carbon/human/spawners.dm]: What's with these changes?

Comment (Naxodile) [reply code/modules/mob/living/carbon/human/spawners.dm]: Cleaned up some of this code I wrote years ago while I was fixing it. Nothing used this and it was a bit pointlessly specific like the rest of the spawner code.

File: maps/away/noctis/noctis_shuttle.dm (hunk 2 above)
Comment (Naxodile) [reply maps/away/noctis/noctis_shuttle.dm]: We should already have generic landmarks around Nerva that are accessible to all shuttles that fit. The shuttle's a pretty strange size, but I imagine you could probably get it to match one of the docking ports.

File: maps/away/noctis/noctis.dm (hunk 1 above)
Comment (Naxodile) [reply maps/away/noctis/noctis.dm]: I sure hope so. This is automatically generated by the in-game circuit exporter with the only alterations being escape characters.

**Pull request merged**